src/App.js

```diff
@@ -20,16 +20,11 @@ function App() {
       })
```

Comment:
Since you aren't chaining any additional functionality after this .then, do you need the `return` here?  Or can you just run setMovies without returning it?

Reply:
Nice job declaring your fetch calls in an apiCalls file.  Good work checking for the ok response and throwing an Error if not.  The next step where would be to use a catch block when you _invoke_ the fetch call to take that thrown Error and do something with it.  

A common approach is having an error piece of state that you update if/when there is an error.  Then you can conditionally render an error message to advise the user.  Best practice is to actually make an error component to display to users - you can use props to make it dynamic so that the same component can display different messaging depending on what is passed as a prop.